crates/astria-sequencer/src/service/mod.rs

```diff
@@ -19,6 +19,7 @@ impl AbciCode {
     pub(crate) const INVALID_PARAMETER: Self = Self(2);
     pub(crate) const INTERNAL_ERROR: Self = Self(3);
     pub(crate) const INVALID_NONCE: Self = Self(4);
+    pub(crate) const INVALID_SIGNATURE: Self = Self(5);
```

Comment:
Does this capture all errors that can occur above?

This should have a clearer mapping from error code to abci code before I'm afraid the error log might not match the code.

Maybe leave it as an internal error for now?

Reply:
yeah i guess there are other errors that could occur like the fields being the wrong length. i'll update